### ISSUE TITLE
Choices & choices from web conflict

### ISSUE THREAD

**Issue by xxxKOTxxx** [surveyjs/survey-library#2827]
I have a question (or UI bug) related to choices-based widget.

If in checkbox/radiobutton widget used both `choices` and `choices from web` data sources, survey shows basic choices on first rendering and after fetching remote data replace them by choices from web. Is it expected behaviour?

I created small reproducing [plunker](https://plnkr.co/edit/WAVpIV6Rs95aDLfT)

Possibly it's better to remove  `choices` if `choices from web` is set? Or just don't render choices while data is loading?

**Comment by tsv2013** [comment]
Yes, they don't work together. We'll discuss what we can do. Right now you can reset `choices` manually if `choicesByUrl` is set.
